### PR TITLE
remove unnecessary ledger.representative() call

### PR DESCRIPTION
**Pull request by zhaoguojie2010**

### CODE REVIEW

**Comment by zhaoguojie2010** [comment]
Why CI failed?

File: rai/secure.cpp [hunk 1]
```diff
@@ -1990,7 +1990,7 @@ class rollback_visitor : public rai::block_visitor
 		rai::account_info info;
 		auto error (ledger.store.account_get (transaction, destination_account, info));
 		assert (!error);
-		ledger.store.representation_add (transaction, ledger.representative (transaction, hash), 0 - amount);
+		ledger.store.representation_add (transaction, representative, 0 - amount);
```
Comment (cryptocode) [inline rai/secure.cpp]: This change basically implies that

`ledger.representative (transaction, block_a.hashables.previous)`

is the same as

`ledger.representative (transaction, block_a.hash ())`

Is that right?

Comment (PlasmaPower) [reply rai/secure.cpp]: Yeah I don't think this change is correct.

Comment (PlasmaPower) [reply rai/secure.cpp]: Yeah I don't think this change is correct.

Comment (zhaoguojie2010) [reply rai/secure.cpp]: rep is chosen by iterating block's previous until open or change block is found, so I think the change is correct.

**Comment by zhaoguojie2010** [comment]
Another change: There's no need to create a election for every recently arrived block.  A election should be created only when a fork a detected, so this should be removed.

**Comment by zhaoguojie2010** [comment]
By the way, how can I create one PR for each commit?

**Comment by PlasmaPower** [comment]
Both of these changes are incorrect. In addition, there are merge conflicts.

**Pull request closed without merging**